### DISCUSSION TITLE
Prisma, Heroku and Now

### DISCUSSION THREAD

**Discussion by tomadimitrie** [prisma/prisma#2492] [Q&A] [answered]
I have my frontend hosted on Zeit Now and my database on Heroku. My question is, what should DATABASE_URL be set to? Heroku docs say that the database url they give is temporary and can change due to security issues. So how can I make sure the url is always the right one?

**Comment by pantharshit00** [comment, marked as answer]
The `DATABASE_URL` should be set to the credentials that heroku gives you. They will only change if you have credential rotation enabled: https://devcenter.heroku.com/articles/heroku-postgresql-credentials#pg-credentials-rotate


If you want to make use of credential rotation, it is a good idea to host your instance on heroku it self as the environment there will have direct access to fresh credentials or otherwise, deploy somewhere where you can execute the heroku cli to grab the new credentials.

**Reply by tomadimitrie** [reply]
thank you!

**Comment by ludralph** [comment]
Hi @tomadimitrie,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
